context/output.go

```diff
@@ -49,6 +49,22 @@ func (output *BeegoOutput) Header(key, val string) {
 	output.Context.ResponseWriter.Header().Set(key, val)
 }
 
+// Flush current output buffer to the client
+func (output *BeegoOutput) Flush() {
+	flusher,ok := output.Context.ResponseWriter.(http.Flusher)
+	if ok {
+		flusher.Flush()
+	}
+}
+
+// Write response content into the buffer directly
+func (output *BeegoOutput) Write(content []byte) {
```

Comment:
There's no need to add this function. `output.Context.ResponseWriter`already has the Write method